### ISSUE TITLE
[Bug] Multi-row tabs overflow is not visible on maximised Fluerial UI

### ISSUE THREAD

**Issue by dominichayesferen** [Floorp-Projects/Floorp#364]
### Attention!

- [X] I undertand that Floorp is NOT supported on Windows 7/8.1 or lower.

### Describe the bug

When maximised and on Fluerial UI, Floorp's multi-row tabs feature only displays the first row of tabs in a manner similar to #

### To Reproduce

1. Switch to Fluerial UI
2. Maximise
3. Create enough tabs to cause a tab overflow 

### Expected behavior

All rows of tabs will be visible

### Screenshots

![image](https://github.com/Floorp-Projects/Floorp/assets/11057934/113437e8-2cac-49ca-af9b-8a534b24f447)
![image](https://github.com/Floorp-Projects/Floorp/assets/11057934/4c084e41-f1f7-440c-9d97-236b1aaa7221)

### OS Info

Feren OS 2023.04

### Floorp Version

11.1.0 (Flathub)

**Comment by surapunoyousei** [comment]
@kaonasi-biwa

**Comment by ToFu-Emon-x-The-Multi-Rower** [comment]
Hello Ablaze Team,

I'm ToFu-Emon, the one of Multi-Row-Tabs Enthusiasts.
Always greatly appreciating your continuous support of the functions for the Multi-row tabs.  
I consider that only your Floorp is the last major web browser currently...

Sorry, to get sidetracked.
If you are talking about the menu items about the multi-row tabs in "Preferences",
the place of them were moved to the other place, "Design" part.

However, the menus and functions of the multi-row tabs still exist and alive.
 
When I installed the first stable release from the 11th train of Floorp,
I considered the multi-row-tabs functions were suddenly deleted, they died unfortunately, I don't know the reason
because I couldn't find the menu items of the multi-row tabs functions at the "General" part in Preferences.
Oh my goodness.

I was absolutely so sad that tears were full in my eyes and also dropped on my cheeks.
After a moment, when I fortunately found the menu in the "Design" part of the "Preferences",
my life became very lucky and happy thanks to your team.

By the way, regarding Vivaldi's multi-row tabs, it is basically same concept and functionality of Window Task Bar when it's at the top of the window.
It is not an unique idea, so we can call it just Browser Window Task Bar with a small portion of eye-catch gimmicks. 
Otherwise, they just made the horizontal tab controllers vertical and allocated the top of browser's window.   

Thanks in advance,
ToFu-Emon
